apps/docs/src/app/contact/page.tsx

```diff
@@ -1,11 +1,19 @@
 import { Heading, Paragraph } from '@govie-react/ds';
+import { config } from '@/lib/config';
 
 export default function ContactPage() {
   return (
     <div>
       <Heading>Contact</Heading>
       <Paragraph>
-        You can contact the design system team via email. Email coming soon.
+        You can contact the design system team using{' '}
+        <a
```

Comment:
could add a TODO to replace with the `Link` component when it supports target (for styling consistency)